tools/shims/package.json

```diff
@@ -27,7 +27,10 @@
         "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/tools/shims"
     },
     "license": "MIT",
-    "sideEffects": false,
+    "sideEffects": [
```

Comment:
Does this affect the sideEffects of consumers of this package?

Reply:
Yes, specifically [webpack tree shaking](https://webpack.js.org/guides/tree-shaking/),

Reply:
Do each of their sideEffects properties need to be updated as well?

Reply:
No, only on the the published module that actually have side effects that are not exported as variables. In this specific case the side effect is the setting / exposing of the global __extends() and __assign() functions. Just by loading the module.